src/components/Footer.js

```diff
@@ -23,85 +22,84 @@ import homeSvg from '../assets/home.svg'
 
 function Footer() {
   const theme = useTheme()
-  const { below } = useViewport()
-  const compactMode = below('large')
 
   return (
     <footer
       css={`
-        flex-shrink: 0;
         width: 100%;
-        padding: ${5 * GU}px ${3 * GU}px;
+        padding: ${1 * GU}px ${1.3 * GU}px;
         background: ${theme.surface};
       `}
     >
-      <Layout paddingBottom={0}>
-        {compactMode ? (
-          <FixedFooter />
-        ) : (
+      <Layout paddingBottom={70}>
+        <FixedFooter />
```

Comment:
Why did we remove the compactMode condition? Note that we only want the Fixed Footer on smaller screen sizes

Reply:
Removed cursor:pointer.
Fixed the href properties to direct correctly.
Still need to configure the compact mode footer tooldbar.